src/schemas/json/swcrc.json

```
@@ -526,6 +526,11 @@
               "type": "boolean",
               "description": "If true, swc generates more efficient code"
             },
+            "keepClassNames": {
+              "type": "boolean",
+              "$comment": "Requires swc@1.2.50 and target es2016 or higher",
+              "description": "Enabling this option will make swc preserve original class names"
```

```suggestion
              "description": "Preserve original class names\nhttps://swc.rs/docs/configuration/compilation#jsckeepclassnames"
```

Optional fix: simplify description.
Optional fix: add link to the documentation.